crates/viewer/re_space_view_graph/src/ui/draw.rs

```diff
@@ -192,24 +192,43 @@ fn draw_arrow(painter: &Painter, tip: Pos2, direction: Vec2, color: Color32) {
 }
 
 /// Draws an edge between two points, optionally with an arrow at the target point.
-fn draw_edge(ui: &mut Ui, points: [Pos2; 2], show_arrow: bool) -> Response {
+pub fn draw_edge(ui: &mut Ui, geometry: &EdgeGeometry, show_arrow: bool) -> Response {
     let fg = ui.style().visuals.text_color();
+    let stroke = Stroke::new(1.0, fg);
 
-    let rect = Rect::from_points(&points);
     let painter = ui.painter();
-    painter.line_segment(points, Stroke::new(1.0, fg));
 
-    // Calculate direction vector from source to target
-    let direction = (points[1] - points[0]).normalized();
+    match geometry.path {
+        PathGeometry::Line { source, target } => {
+            painter.line_segment([source, target], stroke);
+        }
+        PathGeometry::CubicBezier {
+            source,
+            target,
+            control,
+        } => {
+            painter.add(CubicBezierShape {
+                points: [source, control[0], control[1], target],
+                closed: false,
+                fill: Color32::TRANSPARENT,
+                stroke: stroke.into(),
```

Comment:
oh nice, that was easy enough :)

Reply:
Yeah, we will run into limitations though when we want to draw more complex edges (such as in GraphViz layouts), because that's currently not supported by `egui`.

Reply:
This is where kurbo can help. It can convert various things to cubic bezier, and can also linearise everything to polylines.